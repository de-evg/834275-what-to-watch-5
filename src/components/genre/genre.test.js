import React from "react";
import renderer from "react-test-renderer";
import Genre from "./genre";


describe(`Render Genre`, () => {
  it(`Should active Genre render correctly`, () => {
    const tree = renderer
          .create(<Genre
            genre={``}
            isActive={true}
            onGenreFilterChange={() => {}}
          />)
          .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Should unactive Genre render correctly`, () => {
    const tree = renderer
          .create(<Genre
            genre={``}
            isActive={false}
            onGenreFilterChange={() => {}}
          />)
          .toJSON();

    expect(tree).toMatchSnapshot();
  });
});

