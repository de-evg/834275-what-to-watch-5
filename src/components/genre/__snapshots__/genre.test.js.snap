// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Genre Should active Genre render correctly 1`] = `
<li
  className="catalog__genres-item catalog__genres-item--active"
>
  <a
    className="catalog__genres-link"
    href="#"
    id=""
    onClick={[Function]}
  >
    
  </a>
</li>
`;

exports[`Render Genre Should unactive Genre render correctly 1`] = `
<li
  className="catalog__genres-item "
>
  <a
    className="catalog__genres-link"
    href="#"
    id=""
    onClick={[Function]}
  >
    
  </a>
</li>
`;
